docs/examples/make_pymvpa_collection.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+# A little script to prepare PyMVPA demo collection based on data used in
+# tutorials etc
+
+set -eu
+
+echo "I: adding haxby2001"
+# Haxby 2001 -- the main demo dataset
+datalad install-handle http://data.pymvpa.org/datasets/haxby2001/.git
+
+# getting its content, since publishing can't clone the repo, we need to push
+# content to the published handle and therefore we would loose the connection
+# to content, that is not addurl'ed but just available via the original annex.
+# Note: May be therefore use "git annex get . --from=origin" to not unnecessarily
+# get things, that are reachable from the published handle (and any clone of
+# it)? Need to add that option to datalad get.
```

Comment:
1.  Yes -- for autoenabling and "persistence".  So that clones of the clones would immediately know about some distant origin without requiring us to drag/add additional remotes.  Possible gotcha -- not sure how it would behave if original remote disappears/goes offline
2. yes -- it is `type=git` I believe
3. I thought of this somewhat more about "initializing a handle, possibly from an existing annex repo" but indeed it is also related to publishing. and probably we should add those repositories themselves as specialremotes to themselves (based on their git url).... but I wonder if that wouldn't lead to any problem -- yet to check ;-)   But indeed, for some handles we might simply carry somehow list of additional git remotes to be added upon install of that handle, as the sources of data for them
4. well -- I guess then we need to have it implemented ;-)
5. For both of those:  so we could generate  those collections but also to show how we have done it (that there is no magic etc)